pages/destinations/[id].js

```diff
@@ -47,6 +47,7 @@ export default function Destinations({ id, destinationsDB, country, toDosDB }) {
   const [modal, setModal] = useState({ visible: false, name: "" });
   const [loader, setLoader] = useState(false);
   const [showProfile, setShowProfile] = useState(false);
+  const [deleteDestinationId, setDeleteDestinationId] = useState(0);
```

Comment:
Why don't you use null als default value for the useState instead of 0?

Reply:
I declare the state as number. null is not a number and would end up in a type error.